src/pint/observatory/special_locations.py

```diff
@@ -315,8 +330,16 @@ def posvel(self, t, ephem, group=None):
         return geo_posvel + stl_posvel
 
 
-# Need to initialize one of each so that it gets added to the list
-BarycenterObs("barycenter", aliases=["@", "ssb", "bary", "bat"])
-GeocenterObs("geocenter", aliases=["0", "o", "coe", "geo"])
-T2SpacecraftObs("stl_geo", aliases=["STL_GEO"])
-# TODO -- BIPM issue
+def load_special_locations():
+    """Load :class:`~pint.observatory.special_locations.BarycenterObs`, :class:`~pint.observatory.special_locations.GeocenterObs`,
+    and :class:`~pint.observatory.special_locations.T2SpacecraftObs` into observatory registry.
+    """
+    # Need to initialize one of each so that it gets added to the list
+    BarycenterObs("barycenter", aliases=["@", "ssb", "bary", "bat"])
+    GeocenterObs("geocenter", aliases=["0", "o", "coe", "geo"])
+    T2SpacecraftObs("stl_geo", aliases=["STL_GEO"])
+    # TODO -- BIPM issue
```

Comment:
Is this a concern? If they need BIPM TAI to TT data that's awkward because we allow different versions and we want users to be able to use a consistent version throughout PINT.

Reply:
No idea.  This predates me.

Reply:
It probably is a concern but I'm not sure what to do about it. Leave it I guess.

Reply:
There is some complexity here, but discussion belongs on #1305 I think.